visualization/app/codeCharta/model/files/files.helper.ts

```diff
@@ -33,3 +33,7 @@ export function isDeltaState(fileStates: FileState[]): boolean {
 export function isPartialState(fileStates: FileState[]): boolean {
 	return fileStates.some(x => x.selectedAs === FileSelectionState.Partial)
 }
+
+export function getFileState(ccFile: CCFile): FileState {
+	return { file: ccFile, selectedAs: FileSelectionState.None }
```

Comment:
Nit: I would just inline this helper function. It is only used once and is just a tiny single line, thus, there's no actual benefit in abstracting this.